src/combobox/tests/functional/ComboBox.ts

```diff
@@ -0,0 +1,170 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as keys from 'leadfoot/keys';
+import * as css from '../../styles/comboBox.m.css';
+
+const DELAY = 300;
```

Comment:
It may need to be longer in other tests, but 300ms is enough here. I'd rather not make this one any longer than it has to be; if we can lower the others without the tests failing intermittently, then I would be fine with that.